tilebag.js

```diff
@@ -0,0 +1,17 @@
+var TileBag = function() {
+  thos = this;
+};
+
+TileBag.prototype.tiles = {
+  A:9, B:2, C:2, D:4, E:12, F:2, G:3, H:2, I:9, J:1, K:1, L:4, M:2, N:6, O:8, P:2, Q:1, R:6, S:4, T:6, U:4, V:2, W:2, X:1, Y:2, Z:1
+};
+
+TileBag.prototype.drawtiles = function(word) {
+  array = word.toUpperCase().split('');
+  for (letter of array) {
+   thos.tiles[letter] -= 1;
+ };
+  return true
+};
+
```

Comment:
What happens when all the letters are gone? You can check to make sure that the letters remaining are greater than 0 before decrementing. Right now this method will always return true, which does not seem super useful. If your function does not have an explicit return you can `return this` at the end, which allows you to chain methods together. [See this stack overflow for more info.](http://stackoverflow.com/questions/8300844/what-does-return-this-do-within-a-javascript-function).
 I am also not sure how you are using this method in your game, but in a more traditional scrabble game it seems like this method would take the number of tiles to draw as input and return random tiles for the player while decrementing the number left in the bag.